test/e2e/input-addon.test.js

```diff
@@ -9,7 +9,13 @@ describe('vl-input-addon', async () => {
         return vlInputAddonPage.load();
     });
 
-    it("Dummy test om browsers te laten sluiten", () => {
-    	assert.isTrue(true);
+    it('Als gebruiker kan ik de tekst zien van een input addon', async () => {
+        const inputAddon = await vlInputAddonPage.getInputAddon();
+    	await assert.eventually.equal(inputAddon.getText(), '€');
+    });
+
+    it('Als gebruiker kan ik een tooltip zien op een input-addon', async () => {
+        const tooltip = await vlInputAddonPage.getTooltipFromInputAddon();
+        await assert.eventually.equal(tooltip.getText(), 'Euro');
     });
```

Comment:
Ik zou hier een `hover` functie verwachten, dus doet me vermoeden dat de `getText` op de tooltip misschien best anders geïmplementeerd wordt. In dit geval zou ik een lege string verwachten omdat er geen tooltip zichtbaar is.

Reply:
de text is de text toch? de tooltip zou pas "displayed" mogen zijn als je er over hovert, maar de text is de text ...

Reply:
Op zich volg ik Tom wel, ik heb het in de beschrijving wel over 'zien', maar op zich is dit nog verborgen. Ik stel voor van het hoveren buiten scope te houden en de beschrijving wat aan te passen.

Reply:
Ik heb de hover wel eens getest, maar die leek niks te doen. Zowel displayed als text bleven steeds hetzelfde.

Reply:
Ja, inderdaad hier geen probleem. Ik heropen de vl-ui-tooltip om daar een uitbreiding te voorzien om te kunnen testen of de tooltip open staat.